README.md

```diff
@@ -1,231 +1,39 @@
 # Intel® RealSense™ camera driver for GMSL* interface
 
-# D457 MIPI on Jetson AGX Xavier
+# D457 MIPI on Jetson AGX Xavier and Orin 
```

Comment:
Please follow the guidelines sent by mail
Thanks!

Reply:
I tried, hope not missed anything.